packages/bpk-component-content-cards/src/__snapshots__/BpkContentCards-test.js.snap

```diff
@@ -3,9 +3,7 @@
 exports[`Content Card widget renders correctly with 1 card 1`] = `
 <DocumentFragment>
   <div>
-    <h2
-      class="bpk-text bpk-text--body-default bpk-content-cards--header-text"
-    >
+    <h2>
```

Comment:
It's odd that these snaps don't have any `className` included, do you know why?

Reply:
Ah Mungo, sorry, still catching up with onboarding to Backpack, what do you mean by **odd**?

I cannot see any reference to `className` in the snap on `main` either, i.e. there are no class names in the existing snap.

For reference, there are 167 `.snap` files and only 84 have class names in them.

Reply:
Just meaning that we are rendering: 

`<h2 className={STYLES.headerText}>{heading}</h2>`

but within the snapshot (above) we see no `class` on the `h2`.

I _think_ its cause we aren't now using `getClassName`?

Reply:
Reverted to using `getClassName` - DONE 👍